pkgs/applications/audio/deadbeef/default.nix

```
@@ -36,18 +37,21 @@
 
 assert gtk2Support || gtk3Support;
 
-stdenv.mkDerivation rec {
+let
+  version = "1.9.1";
```

And maybe also get rid of the `with` statement below:

```suggestion
  version = "1.9.1";

  inherit (lib) optionals;
```

---

Yeah, I guess we could drop GTK 2 support altogether since GTK 2 is no longer maintained. But probably best to keep that separate.

But I have definitely seen people allergic to systemd and PulseAudio.